examples/cloud-operations/workload-identity-federation/azure.tf

```diff
@@ -0,0 +1,177 @@
+locals {
+  username    = "azureuser"
+  app_name    = "test-app"
+  custom_data = <<CUSTOM_DATA
+#!/bin/bash
+cat <<EOF > /home/${local.username}/credentials.json
+{
```

Comment:
I would put this into an external file and then call `templatefile()`

Reply:
done